qutip/core/cy/qobjevo.pyx

```diff
@@ -482,7 +482,15 @@ cdef class QobjEvo:
                     f"Desired feedback {key} is not available for the {solver}."
                 )
             new_args[key] = solvers_feeds[feed]
-        self.arguments(**new_args)
+
+        if new_args:
+            cache = []
+            self.elements = [
+                element.replace_arguments(new_args, cache=cache)
+                for element in self.elements
+            ]
+
+
```

Comment:
These extra blank lines should be removed but GitHub's suggestion syntax doesn't render the removal of them nicely.

Reply:
```suggestion
```